ntoskrnl/se/semgr.c

```diff
@@ -122,6 +124,14 @@ SepInitializationPhase0(VOID)
     ObInitializeFastReference(&PsGetCurrentProcess()->Token, NULL);
     ObInitializeFastReference(&PsGetCurrentProcess()->Token,
                               SepCreateSystemProcessToken());
+
+    /* Initialise the anonymous logon tokens */
+    SeAnonymousLogonToken = SepCreateSystemAnonymousLogonToken();
+    if (!SeAnonymousLogonToken) return FALSE;
```

Comment:
```suggestion
    if (!SeAnonymousLogonToken)
        return FALSE;
```
We don't allow that with our current coding style (you are touching a damn old code!)

Reply:
I wrote this chunk of code that way for the sake of preserving the existing code style of the function. But fine.